src/tools/migrations/steps/steps/20251028134533-step-generic-replace-ref-id-with-uuid.ts

```diff
@@ -143,7 +148,32 @@ export default async (): Promise<void> => {
 
     await client.none(`alter table ${schemaName}.row drop column if exists table_id`)
 
-    // Regenerate metadata cache (sections, sectionsMetadata, rows)
+    // ============= col.row_id -> row.row_uuid
+
+    await client.none(`alter table ${schemaName}.col add column if not exists row_uuid uuid`)
+
+    await client.none(`
+      update ${schemaName}.col c
+        set row_uuid = r.uuid
+      from ${schemaName}.row r
+      where c.row_id = r.id
+    `)
+
+    await client.none(`
+      alter table ${schemaName}.col
+        alter column row_uuid set not null,
+        add constraint col_row_uuid_fk
+          foreign key (row_uuid)
+          references ${schemaName}.row (uuid)
+          on update cascade
+          on delete cascade
+    `)
+
+    await client.none(`alter table ${schemaName}.col drop column if exists row_id`)
+
     await CacheController.generateMetadata({ assessment }, client)
+
+    // recreate dropped views
+    await Promise.all(assessment.cycles.map((cycle) => TableData.refreshViews({ assessment, cycle }, client)))
```

Comment:
This before cache generation

Reply:
Shouldn't affect? We are generating metadata cache? I was 50/50 if we should also generate data cache after generating the views, but nothing should change in this case?

Reply:
sure, updated